### PR TITLE
WIP: Update guide_for_teachers.md

### PR DESCRIPTION
**Pull request by katrite**
## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) of your branch.

## Checklist before requesting a review

- [ ] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [ ] My pull request passes the automated tests. (If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.)
- [ ] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

**Comment by rkronberg** [comment]
Ping @tourunen :) Would you have time to check this?

File: docs/cloud/csc_notebooks/guide_for_teachers.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ If you cannot find a suitable image for your intended application, you will need
 Requirements: 
 
 * A computer to create the Docker image, it should have [Docker](https://www.docker.com/) installed. In general Linux/Mac computer is recommended. In Windows likely admin rights are needed and using Docker might be challenging.
-* A place to upload the Docker image, for example DockerHub or Quay.io.
```
Comment (tourunen) [inline docs/cloud/csc_notebooks/guide_for_teachers.md:92]: IMO DockerHub is a valid place to host images, we should leave it.

File: docs/cloud/csc_notebooks/guide_for_teachers.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Instead of a course you can also use CSC Notebooks for collaboration. The workfl
 
 * The easiest is to use an existing Docker image, look these repositories for suitable images:
     * [Docker image sources in notebooks-public-images repository.](https://github.com/CSCfi/notebooks-public-images)
-    * [Rocker images](https://hub.docker.com/u/rocker) for different RStudio set-ups.
```
Comment (tourunen) [inline docs/cloud/csc_notebooks/guide_for_teachers.md:29]: To me it looks like Rocker project still hosts images on DockerHub as the primary source, and the other provided registry is GitHub container registry. 

https://rocker-project.org/images/versioned/r-ver.html#quick-reference

File: docs/cloud/csc_notebooks/guide_for_teachers.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ For other package management systems, adjust the last `RUN` command accordingly.
     
 #### RStudio example
     
-To build custom R images, you do not need to start from scratch. Many pre-built R images are already available in docker registries. Especially, the [rocker project](https://github.com/rocker-org/rocker-versioned2) contains a large set of images with various configurations provided in [DockerHub](https://hub.docker.com/u/rocker/). You can therefore start with one of these pre-existing images.
```
Comment (tourunen) [inline docs/cloud/csc_notebooks/guide_for_teachers.md:133]: I think the old text is still fine.

**Review by tourunen — CHANGES_REQUESTED** [review]

**Comment by rkronberg** [comment]
Closing this PR based on comments by @tourunen (old text still ok and relevant).

**Pull request closed without merging**